.github/workflows/actions/integration-test/action.yml

```diff
@@ -0,0 +1,30 @@
+name: Run Integration tests
+
+inputs:
+  monitoring_instance_name:
+    description: The name of monitoring instance
+    required: true
+
+runs:
+  using: composite
+
+  steps:
+    - name: test prometheus
+      uses: wei/curl@master
```

Comment:
Is curl not natively on the agents?

Reply:
I had a chat with @merrilmathew . This action is quite convenient as it automatically fails if the URL fails

Reply:
We would need to write the logic ourselves to fail the step based on return code. This action is pretty neat because it will fail if there is http error.